setup.py

```diff
@@ -244,6 +244,12 @@ def get_data_files():
     'webpdf': [
         pyppeteer_req
     ],
+    'qtpdf': [
```

Comment:
As with `pyppeteer`, this likely needs to be added to `test`.

Another approach, as we've done on ipython, is to move all of these to `setup.cfg` and use the barely-documented string replacement technique: https://github.com/ipython/ipython/blob/master/setup.cfg#L78